lib/bcdice/game_system/HunterTheReckoning5th.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+module BCDice
+  module GameSystem
+    class HunterTheReckoning5th < Base
+      # ゲームシステムの識別子
+      ID = 'HunterTheReckoning5th'
+
+      # ゲームシステム名
+      NAME = 'Hunter: The Reckoning 5th Edition'
+
+      # ゲームシステム名の読みがな
+      SORT_KEY = 'はんあたされこにんく5'
+
+      # ダイスボットの使い方
+      HELP_MESSAGE = <<~MESSAGETEXT
+        ・判定コマンド(nHRFx+x)
+          注意：難易度は必要成功数を表す
+
+          難易度指定：成功数のカウント、判定成功と失敗、Critical処理、Critical Winのチェックを行う
+                     （Desperationダイスがある場合）OverreachとDespairの発生チェックを行う
+          例) (難易度)HRF(通常ダイス)+(Desperationダイス)
+              (難易度)HRF(通常ダイス)
+
+          難易度省略：成功数のカウント、判定失敗、Critical処理、（Desperationダイスがある場合）Despairチェックを行う
+                      判定成功、Overreachのチェックを行わない
+                      Critical Win、（Desperationダイスがある場合）Despair、Overreachのヒントを出力
+          例) HRF(通常ダイス)+(Desperationダイス)
+              HRF(通常ダイス)
+
+          難易度0指定：全てのチェックを行わない
+          例) 0HRF(通常ダイス)+(Desperationダイス)
+              0HRF(通常ダイス)
+
+      MESSAGETEXT
+
+      DIFFICULTY_INDEX   = 1
+      DICE_POOL_INDEX    = 3
+      DESPERATION_DICE_INDEX = 5
+
+      # 難易度に指定可能な特殊値
+      NOT_CHECK_SUCCESS = -1 # 判定成功にかかわるチェックを行わない(判定失敗に関わるチェックは行う)
+
+      # ダイスボットで使用するコマンドを配列で列挙する
+      register_prefix('\d*HRF')
+
+      def eval_game_system_specific_command(command)
+        m = /\A(\d+)?(HRF)(\d+)(\+(\d+))?/.match(command)
```

Comment:
```suggestion
        m = /\A(\d+)?(HRF)(\d+)(\+(\d+))?$/.match(command)
```
コマンドを正規表現でパースするときには、完全一致で動作するように記述してください。

Reply:
ご指摘、ありがとうございます。修正いたします。
V:tM5thの方でも同じようになっていたので、近日中にそちらも修正いたします。